hardware_layer/hardware_arduino/src/MS5837.h

```diff
@@ -1,10 +1,10 @@
 /* Blue Robotics Arduino MS5837-30BA Pressure/Temperature Sensor Library
```

Comment:
I didn't know about it until now. I asked @pkhrag and he told me to run format_cpp.sh and then checkout the changes in the ip nodes. I'll take care the next time.